Modules/Core/Support/PdfHelper.php

```
@@ -52,14 +52,19 @@ public static function discount_global_print_in_pdf($obj, $show_item_discounts,
      */
     public static function generate_invoice_pdf($invoice_id, $stream = true, $invoice_template = null, $is_guest = null)
     {
-        $invoice = \Modules\Invoices\Models\Invoice::with(['client', 'user'])->find($invoice_id);
+        $invoiceService = app(\Modules\Invoices\Services\InvoiceService::class);
+        $paymentMethodService = app(\Modules\Payments\Services\PaymentMethodService::class);
+        $invoiceItemService = app(\Modules\Invoices\Services\InvoiceItemService::class);
+        $invoiceTaxRateService = app(\Modules\Invoices\Services\InvoiceTaxRateService::class);
+        
+        $invoice = $invoiceService->findWithRelations($invoice_id, ['client', 'user']);
```

PHPMD (2.15.0)

The variable $invoice_id is not named in camelCase. (undefined)

(CamelCaseVariableName)